core/expression/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 authors = ["GoRules Team <bot@gorules.io>"]
-description = "Zen expression language virtual machine"
-name = "zen-vm"
+description = "Zen Expression Language"
+name = "zen-expression"
 license = "MIT"
 version = "0.2.1"
```

Comment:
We should set this to "0.1.0"

Reply:
We need to follow versions as engine package